onnxruntime/core/providers/webnn/builders/impl/conv_op_builder.cc

```diff
@@ -276,7 +277,12 @@ Status ConvOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder, const N
     if (!is_nhwc || !is_constant_weight) {
       // The weight_shape has been appended 1's, reshape weight operand.
       std::vector<uint32_t> new_shape = GetVecUint32FromVecInt64(weight_shape);
-      filter = model_builder.GetBuilder().call<emscripten::val>("reshape", filter, emscripten::val::array(new_shape));
+      emscripten::val reshape_options = emscripten::val::object();
+      reshape_options.set("label", node.Name() + "_reshape_filter");
```

Comment:
I was going to ask if snake case is really appropriate for a camelCase API (WebNN is camelCase), but then these labels are really ORT-specific (which followed Python naming for locals rather than Javascript) because they're inserted extra tensors. So resolving my own comment.